services/workflows-service/scripts/clean.ts

```diff
@@ -18,7 +18,7 @@ async function clean() {
   const tables = await getTables(prisma);
   const types = await getTypes(prisma);
   await dropTables(prisma, tables);
-  await dropTypes(prisma, types);
+  await dropTypes(prisma, types.sort());
```

Comment:
Cleaning the db sometimes fails due to the order in which the types are deleted.

Reply:
I don't think it's a good practice to boyscout in big PRs.
You know I'm totally for boyscouting. Cleaning and refactoring unrelated things as part of your tasks is a good practice IMO. But here it's just overwhelming.
You don't have to revert it now, but for next times I think we should try not to do it.

Reply:
Should I add the sort -> run db:clean -> remove the sort? The error kept happening in this PR. :slightly_smiling_face: